Index: app/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/build
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/.gitignore b/app/.gitignore
--- a/app/.gitignore	(revision 95790da88c01667cbd1a51cb0b394e516c580251)
+++ b/app/.gitignore	(date 1744330068674)
@@ -1,1 +1,0 @@
-/build
\ No newline at end of file
